.github/workflows/ci.yml

```diff
@@ -13,6 +13,8 @@ jobs:
       - uses: actions/checkout@v2
 
       - uses: purescript-contrib/setup-purescript@main
+        with:
+          purescript: "unstable"
 
       - uses: actions/setup-node@v1
```

Comment:
Can we fix the Node version? `setup-node@v2` and `node-version: "14"`?

Reply:
Done.